doc/ModuleStartupOrder.md

```diff
@@ -58,7 +58,7 @@ The following sample deployment manifest illustrates how startupOrder values of
           "edgeAgent": {
             "type": "docker",
             "settings": {
-              "image": "mcr.microsoft.com/azureiotedge-agent:1.0",
+              "image": "mcr.microsoft.com/azureiotedge-agent:1.0.10",
```

Comment:
If the release process is done correctly, the docker image tag `1.0.10.1` and `1.0` should be synonymous. This change is uncessary.